### DISCUSSION TITLE
(Mobile) Sorting Menu Options

### DISCUSSION THREAD

**Discussion by agithony** [dailydotdev/daily#435] [Feature Requests]
On the mobile app, when you enable "Show feed sorting menu" in settings, the sorting menu appears. However, the drop-down options are limited.

For Popular, we only have 2 options:
1. Recommended
2. Date

For Most Upvoted, we have 3 options:
1. Last week
2. Last month
3. Last year

For Best Discussions, we have 0 options.

It would be nice to have:
1. "Last day option" for Most Upvoted. Since this helps promote more usage (I don't want to keep seeing the same articles each week)
2. "Last (x)" options for Popular.
3. Some options for Best Discussions.

**Comment by idoshamun** [comment]
Hi @anthonyjdella,

Thank you for the suggestion.
Currently, we don't plan on introducing more sorting options but this might be relevant in the future.
By the way, the desktop version has the same sorting options.

**Reply by agithony** [reply]
If I want to implement this myself, any tips in the codebase? I haven't looked at it yet.

**Reply by idoshamun** [reply]
Thank you so much for your positive will! But unfortunately, this is not something we're looking to implement either by our core team or community contribution. I hope we will have some time to add some issues to let the community work on their contribution 💪
